hindi/priyanka-chopra

```diff
@@ -0,0 +1 @@
+Priyanka Chopra is an Indian film actress and singer, and the winner of the Miss World pageant of 2000. (https://en.wikipedia.org/wiki/Priyanka_Chopra)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.